09-mapping.Rmd

```diff
@@ -727,7 +763,7 @@ g1
 ```
 
 An advantage of **ggplot2** is that it has a strong user-community and many add-on packages.
-Good additional resources can be found in the [ggplot2-book](https://github.com/hadley/ggplot2-book) open source book on the subject, plus descriptions of the multitude of '**gg**packages' such as **ggrepel** and **tidygraph**.
+Good additional resources can be found in the open source [ggplot2-book](https://github.com/hadley/ggplot2-book) and in the descriptions of the multitude of '**gg**packages' such as **ggrepel** and **tidygraph**.
```

Comment:
is it really open source?

Reply:
@Robinlovelace

Reply:
Yes - I linked to the source code (of version II of the book, version I is not).